plugins/modules/eos_bgp_global.py

```diff
@@ -0,0 +1,2302 @@
+            shutdown:
+              description: When True, shut down BGP.
+              type: bool
+            timers:
+              description: Timers.
+              type: dict
+              suboptions:
+                keepalive:
+                  description: Keep Alive Interval in secs.
+                  type: int
+                holdtime:
+                  description: Hold time in secs.
+                  type: int
+            ucmp:
+              description: Configure unequal cost multipathing.
+              type: dict
+              suboptions:
+                fec:
+                  description: Configure UCMP fec utilization threshold.
+                  type: dict
+                  suboptions:
+                    trigger:
+                      description: UCMP fec utilization too high threshold.
+                      type: int
+                    clear:
+                      description: UCMP FEC utilization Clear thresholds.
+                      type: int
+                link_bandwidth:
+                  description: Configure link-bandwidth propagation delay.
+                  type: dict
+                  suboptions:
+                    mode:
+                      description: UCMP link bandwidth mode
+                      type: str
+                      choices: ['encoding_weighted', 'recursive']
+                    update_delay:
+                      description: Link Bandwidth Advertisement delay.
+                      type: int
+                mode:
+                  description: UCMP mode.
+                  type: dict
+                  suboptions:
+                    set:
+                      description: If True, ucmp mode is set to 1.
+                      type: bool
+                    nexthops:
+                      description: Value for total number UCMP nexthops.
+                      type: int
+            update:
+              description: Configure BGP update generation.
+              type: dict
+              suboptions:
+                wait_for:
+                  description: wait for options before converge or synchronize.
+                  type: str
+                  choices: ['wait_for_convergence', 'wait_install']
+                batch_size:
+                  description: batch size for FIB route acknowledgements.
+                  type: int
+    running_config:
+      description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the EOS device by
+        executing the command B(show running-config | section bgp).
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into Ansible structured data as per the resource module's argspec
+        and the value is then returned in the I(parsed) key within the result.
+      type: str
+    state:
+      description:
+      - The state the configuration should be left in.
```

Comment:
Can we add some information about the purged state and the behavior of deleted and replaced when AF is present? Example https://github.com/ansible-collections/cisco.nxos/pull/225/files#diff-3499e660b6220f3413f95f9d0cd654a6daa287ece516ac3eb78415ae0b737780R469-R479